quintilesims/d.ims.io
# Add CRUD operations for images to d.ims.io

**What does this pull request do?**
Adds two new endpoints that deal with images:
- `/images`
    - `GET` lists every image in the registry (similar to the old `catalog` endpoint)
- `/repository/:owner/:name/image/:tag`
    - `GET` describes an image with the specified `:tag`
    - `DELETE` deletes an image with the specified `:tag`


**How should this be tested?**
`go run main.go` locally and then run some CURLs to confirm that the endpoints work:
- `curl -s -u [USERNAME] localhost:9090/images` etc.

Probably also check up on Swagger, as much as it'll work locally.


**Checklist**
- ~Smoke tests (if applicable)~
- ~System tests (if applicable)~
- [x] Documentation _(via Swagger UI)_


closes #42
